internal/xds/translator/testdata/out/xds-ir/multiple-listeners-same-port-with-different-filters.routes.yaml

```diff
@@ -8,11 +8,6 @@
     - match:
         pathSeparatedPrefix: /foo1
       name: httproute/default/httproute-1/rule/0/match/0/www_foo_com
-      responseHeadersToAdd:
```

Comment:
why did this get removed ?

Reply:
The original test input just doesn't make sense, so I removed the http3 settings in that file.

internal/xds/translator/testdata/in/xds-ir/multiple-listeners-same-port-with-different-filters.yaml

```
  - name: default/gateway-1/http
    address: 0.0.0.0
    hostnames:
      - 'www.foo.com'
    isHTTP2: false
    http3:
      quicPort: 443
    path:
      escapedSlashesAction: UnescapeAndRedirect
      mergeSlashes: true
    port: 10080
```